README.md

```diff
@@ -817,6 +824,13 @@ params.ReleaseNotes = "New Release notes"
 params.ReleaseNotesSyntax = "plain_text"
 params.TargetProps = "key1=val1;key2=val2,val3"
 
+// Artifact path mapping:
+// After distribution, the artifact "source-repo/a/123.zip" destination in the edge node will be "target-repo/a-123.zip".
+// The spec's pattern is a wildcard with capturing groups surrounded by parenthesis. 
+// Respectfully, the spec target's replacement groups surrounded by curly brackets.
```

Comment:
Let's modify the description to be:

// The Target property defines the target path in the edge node, and can include replaceable in the form of {1}, {2}, ...
// Read more about it in the above "Creating a Release Bundle" section.